cfe_internal/enterprise/templates/httpd.conf.mustache

```diff
@@ -215,14 +215,14 @@ LogLevel warn
     object-src 'none'; \
     frame-src 'self'; \
     child-src 'self'; \
-    img-src 'self' avatars.githubusercontent.com badges.gitter.im fonts.gstatic.com kiwiirc.com raw.githubusercontent.com; \
+    img-src 'self' avatars.githubusercontent.com badges.gitter.im fonts.gstatic.com kiwiirc.com raw.githubusercontent.com data:; \
     font-src 'self' data: fonts.googleapis.com fonts.gstatic.com; \
     connect-src 'self' fonts.gstatic.com fonts.googleapis.com; \
     manifest-src 'self'; \
     base-uri 'self'; \
     form-action 'self'; \
     media-src 'self'; \
-    worker-src 'self';"
+    worker-src 'self' blob:"
```

Comment:
I added this change to my running hub and at least things did not break obviously.

Reply:
right, look in js console in host info cmdb show json and policy analyzer, that's where we use ACE editor.